cpu/cc2538/include/cpu_conf.h

```diff
@@ -43,6 +43,32 @@ extern "C" {
 #define CPU_HAS_BITBAND                 (1)
 /** @} */
 
+
+/**
+ * @brief   Flash page configuration
+ * @{
+ */
+#define FLASHPAGE_SIZE          (2048U)
+/* Last page holds the CCA region for safety this will not be allowed
+   to be erased */
+#ifndef FLASHPAGE_CC2538_CCA_PAGE
```

Comment:
How about

```suggestion
#ifndef FLASHPAGE_CC2538_USE_CCA_PAGE
```

Reply:
What does CCA stand for?

Reply:
Flash Customer Configuration Area (CCA)

Reply:
It contains the entry point for the application as well as flashpage lock bits, bootloader backdoor acces.